### PR TITLE
Karma

### PR DESCRIPTION
**Pull request by eddiemoore**
Fixing #49

### CODE REVIEW

**Comment by cbas** [comment]
Don't check in `vendor/sinon.js`

**Comment by eddiemoore** [comment]
It's for webpack. something wrong with webpack and using sinon. so need to use the compiled one.

**Comment by eddiemoore** [comment]
@cbas Fixed it

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Run `npm test`
 
 ### Loading Ferry
 ```js
-import Ferry, {Packet, PacketTypes, FerryEvent} from 'migme-ferry'
+import Ferry, {Packet, PacketType, FerryEvent} from 'migme-ferry'
```
Comment (evantage) [inline README.md]: test

**Pull request merged**